docs/csharp/debugger-settings.md

```diff
@@ -10,11 +10,112 @@ MetaDescription: Configuring C# debugging
 
 You can configure the C# debugger in Visual Studio Code with a `launch.json`, `launchSettings.json`, or your user `settings.json` file.
 
-Below are common options you may want to change while debugging.
+## Walk through: setting command line arguments
```

Comment:
Should this be `Walkthrough` (without a space)?

Reply:
Yes